chapter08/compose.override.yaml

```diff
@@ -53,14 +53,12 @@ services:
     user: "0:0"
 
   movielens:
-    build: docker/movielens-api
```

Comment:
I meant to us eit the other way around. So keep the build and lost the image

Reply:
@ismaelcv Could you have a look at this one still? If the doc review is done we can merge this one too